src/pages/docs/creators/javascript/examples/tools/requestor-uninstall.md

```diff
@@ -0,0 +1,124 @@
+---
+title: How to remove Yagna?
+description: Step-by-step guide to removing Yagna components, configs, and data for Requestors.
+type: noindex
+---
+
+# Golem Requestor uninstallation guide
+
+{% alert level="warning" %}
+Before removing Yagna please make sure you have backup-up your private key to the address where you have GLMs. You can find instructions [here](/docs/creators/javascript/examples/tools/golem-wallet-backup).
```

Comment:
Before removing Yagna, please make sure **you have backed up** your private key.
```suggestion
Before removing Yagna, please make sure you have backed up your private key. You can find the instructions [here](/docs/creators/javascript/examples/tools/golem-wallet-backup).
```

Backing up to a place where you have GLM's doesn't make sense.